docs/Config.md

```diff
@@ -82,6 +82,7 @@ git:
   autoRefresh: true
   branchLogCmd: 'git log --graph --color=always --abbrev-commit --decorate --date=relative --pretty=medium {{branchName}} --'
   allBranchesLogCmd: 'git log --graph --all --color=always --abbrev-commit --decorate --date=relative  --pretty=medium'
+  sortRemoteBranchesByLastCommit: false
```

Comment:
Sorting can be done in all sorts of manners, I'd rather not have a `bool` key for sorting by one parameter.

It can be sorted by `lastCommit`, `alphabetical`, whatnot, and in ascending or descending order.

Reply:
Agreed about using an enum. I'm not sure that we need an ascending/descending thing: I'm imagining we just want alphabetical (implicitly ascending), lastCommit (implicitly descending), and whatever else we think of later will probably have some obvious direction users would want.